api/utils/index.js

```diff
@@ -0,0 +1,189 @@
+const { isEmpty, isUndefined } = require("lodash");
+
```

Comment:
I think it would be better to write your own implementation of these functions instead of using lodash just for them